src/presentation/http/router/noteList.ts

```diff
@@ -0,0 +1,64 @@
+import type { FastifyPluginCallback } from 'fastify';
+import type NoteListService from '@domain/service/noteList';
```

Comment:
```suggestion
import type NoteListService from '@domain/service/noteList.js';
```

Reply:
Rewrote the code, this import was removed